app/javascript/lib/service-worker-cache.js

```diff
@@ -28,7 +28,7 @@ export async function fetchServiceWorkerCache(url, init) {
     isHit = false
   }
   const hitOrMiss = isHit ? 'hit' : 'miss'
-  console.debug(`Service worker cache ${hitOrMiss} for SCP API fetch of URL: ${url}`)
+  console.log(`Service worker cache ${hitOrMiss} for SCP API fetch of URL: ${url}`)
```

Comment:
How come you're switching to `console.log` instead here?

Reply:
I've been surprised not to see requests in my DevTools Network panel because they're fetched via service worker cache, and I also typically don't think to switch my Console panel to debug mode, so I think the benefit to debuggability of seeing these messages by default would outweigh the cost of extra logging.  These logs are only written in development.

If you prefer `debug` over `log`, let me know and I'll switch it back!

Reply:
ah okay that works, thanks for the context! No preference from me for `debug` or `log` at all, I just wanted to make sure I wasn't missing a convention or anything I ought to be following as well